src/main/java/com/exadel/movie_new/dao/MovieDao.java

```diff
@@ -0,0 +1,23 @@
+package com.exadel.movie_new.dao;
+
+import com.exadel.movie_new.model.Movie;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+public interface MovieDao {
+
+    List<Movie> readJsonData();
+    void writeJsonData(List<Movie> movieData);
+
+    Movie addMovie(Movie movie);
+
+    List<Movie> getAllMovies();
+
+    Movie getMovie(String movieId);

```

Comment:
as you have MovieDao - you can name your methods more simpler: get, getAll, delete... Because it's oblivious that all operations will be with  movie(s)